src/plugins/intel_cpu/src/ngraph_transformations/swap_convert_transpose.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "swap_convert_transpose.hpp"
+
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+NGRAPH_RTTI_DEFINITION(MKLDNNPlugin::SwapConvertTranspose, "SwapConvertTranspose", 0);
+
+MKLDNNPlugin::SwapConvertTranspose::SwapConvertTranspose() {
+    ngraph::element::TypeVector param_precisions{ ngraph::element::i8, ngraph::element::u8 };
+    auto input_m = ngraph::pattern::wrap_type<ngraph::opset1::Parameter>(ngraph::pattern::type_matches_any(param_precisions));
+    auto convert_m = ngraph::pattern::wrap_type<ngraph::opset1::Convert>({input_m}, ngraph::pattern::type_matches(ngraph::element::f32));
+    auto transpose_m = ngraph::pattern::wrap_type<ngraph::opset1::Transpose>({convert_m, ngraph::pattern::any_input()});
+
+    ngraph::matcher_pass_callback callback = [=](ngraph::pattern::Matcher& m) {
+        // Swap
+        // Input -> [i8/u8] -> Convert -> [f32] -> Transpose -> [f32]
+        // to
+        // Input -> [i8/u8] -> Transpose -> [i8/u8] -> Convert -> [f32]
+        const auto& pattern_map = m.get_pattern_value_map();
+        auto convert = pattern_map.at(convert_m).get_node_shared_ptr();
+        auto transpose = pattern_map.at(transpose_m).get_node_shared_ptr();
+        if (convert != nullptr && ov::is_type<ngraph::opset1::Convert>(convert)) {
+            if (!(convert->get_input_element_type(0) == ngraph::element::i8 ||
+                convert->get_input_element_type(0) == ngraph::element::u8) ||
+                convert->get_output_element_type(0) != ngraph::element::f32) {
+                return false;
+            }
+            auto input = convert->get_input_node_shared_ptr(0);
+            if (input != nullptr && ngraph::op::is_parameter(input)) {
```

Comment:
We don't need to check precisions and nodes on nullptr here. We have already checked it in the matcher. I propose to remove it.
```suggestion
    auto input = pattern_map.at(input_m).get_node_shared_ptr();
```